adr/2024-11-2-assertion-format.md

```diff
@@ -1,78 +1,101 @@
-# [short title of solved problem and solution]
+# `format` as an assertion
 
-* Status: proposed
+- Status: proposed
   <!-- will update below to only those who participated in the vote -->
-* Deciders: @gregsdennis @jdesrosiers @julian @jviotti @mwadams @karenetheridge @relequestual
-* Date: 2024-11-02
-* Technical Story: https://github.com/json-schema-org/json-schema-spec/issues/1520
-* Voting issue: https://github.com/json-schema-org/TSC/issues/19
-    For - @gregsdennis @jdesrosiers @jviotti @mwadams @karenetheridge
-    Neutral - @relequestual
-    Against - @julian
+- Deciders: @gregsdennis @jdesrosiers @julian @jviotti @mwadams @karenetheridge
+  @relequestual
+- Date: 2024-11-02
+- Technical Story: <https://github.com/json-schema-org/json-schema-spec/issues/1520>
```

Comment:
This issue is "make format validate by default", which would seem to be the wrong one?